util/tlgen/item.py

```diff
@@ -58,6 +58,12 @@ class Node:
     req_fifo_pass = True
     rsp_fifo_pass = True
 
+    # FIFO depth option. default is 1
+    # If pipeline is false or req/rsp_fifo_pass are true, this field has no meaning
```

Comment:
Why does the depth have no meaning if the either of the pass options are true? I guess it's just a limitation of the current implementation, right?

Reply:
- If req/rsp_fifo_pass is set, the requests can pass through immediately. I guess in this case we could also make the depth variable. on the other hand, the requests+responses can be buffered by the receiving side if pass-through is set.
- if pipeline is false, the depth is set to 0